root/defaults/nginx/authelia-location.conf.sample

```diff
@@ -1,10 +1,15 @@
-## Version 2023/04/27 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/authelia-location.conf.sample
+## Version 2024/03/14 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/authelia-location.conf.sample
 # Make sure that your authelia container is in the same user defined bridge network and is named authelia
 # Rename /config/nginx/proxy-confs/authelia.subdomain.conf.sample to /config/nginx/proxy-confs/authelia.subdomain.conf
-# Make sure that the authelia configuration.yml has 'path: "authelia"' defined
+# For authelia 4.37 and below, make sure that the authelia configuration.yml has 'path: "authelia"' defined
+# For authelia 4.38 and above, make sure that the authelia configuration.yml has 'address: "tcp://0.0.0.0:9091/authelia"' defined
```

Comment:
fwiw, it can be `tcp://:9091/` which is also what users will see if they check the upstream docs, therefore i suggest using this.

Reply:
Resolved in https://github.com/linuxserver/docker-swag/pull/465/commits/c63b437e1f40acb19e7678cbc962d01c10879b5d